tests/rustdoc-gui/help-page.goml

```diff
@@ -66,5 +66,5 @@ go-to: "file://" + |DOC_PATH| + "/test_docs/index.html"
 set-window-size: (1000, 1000) // Popover only appears when the screen width is >700px.
 assert-false: "#help"
 click: "#help-button > a"
-click: "//*[@id='help']//a[text()='the rustdoc book']"
+click: "//*[@id='help']//a[normalize-space()='the rustdoc book']"
```

Comment:
Is it really worth to add `wbr` for this sentence? Even without it, it works fine (can't get any smaller, after that we switch to mobile and the popover doesn't exist there):

![image](https://github.com/user-attachments/assets/a671093c-0973-49be-a421-ccf1be072afe)